component/file_cache/file_cache.go

```diff
@@ -890,8 +894,14 @@ func (fc *FileCache) ReadFile(options internal.ReadFileOptions) ([]byte, error)
 func (fc *FileCache) ReadInBuffer(options internal.ReadInBufferOptions) (int, error) {
```

Comment:
So, with the new changes, how do we update a cache hit on a read call? or is this just gonna be completely skipped? if so how are we evicting? purely on expiry time?